packages/app/src/cli/services/dev.ts

```diff
@@ -107,15 +98,12 @@ async function dev(options: DevOptions) {
     localApp = await installAppDependencies(localApp)
   }
 
-  const frontendConfig = localApp.webs.find(({configuration}) => configuration.type === WebType.Frontend)
-  const backendConfig = localApp.webs.find(({configuration}) => configuration.type === WebType.Backend)
+  const backendConfig = localApp.webs.find((web) => isWebType(web, WebType.Backend))
```

Comment:
Should we support multiple configs here, since we could technically have multiple backends?

Reply:
I think only one can be The Backend which the frontend iframe accesses via the `BACKEND_PORT` env var. If an app has multiple backends, these can use the `background` role. If the processes need to communicate, they should set explicit ports and hardcode in the dependent processes.

Basically the new rules are:

1. Every app has at least 1 role
2. There must be a `frontend`
3. There can only be 1 `frontend` (embedded app) and 1 `backend` (used for `BACKEND_PORT`). Unlimited webs can be `background` (generic role with no special behavior for now)
4. The same app can be both `frontend` and `backend`